apps/base-docs/docs/pages/chain/troubleshooting.mdx

```diff
@@ -0,0 +1,13 @@
+---
+title: Troubleshooting
+slug: /troubleshooting
+description: Information on how to troubleshooting a Base node
+author: hndvx
+---
+
+# Troubleshooting
```

Comment:
@wbnns do you have common troubleshooting steps listed somewhere?

Reply:
Added 👍